### PR TITLE
[Merged by Bors] - doc(Topology/../DomMulAct): improve docs

### PR DESCRIPTION
**Pull request by urkud**
---
Cleaning up old branches.

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Topology/Algebra/Constructions/DomMulAct.lean [hunk 1]
```diff
@@ -7,11 +7,17 @@ import Mathlib.Topology.Homeomorph
 import Mathlib.GroupTheory.GroupAction.DomAct.Basic
 
 /-!
-# Topological structure on `DomMulAct _`
+# Topological space structure on `Mᵈᵐᵃ` and `Mᵃᵐᵃ`
```
Comment (j-loreaux) [inline Mathlib/Topology/Algebra/Constructions/DomMulAct.lean:10]:
```suggestion
# Topological space structure on `Mᵈᵐᵃ` and `Mᵈᵃᵃ`
```

File: Mathlib/Topology/Algebra/Constructions/DomMulAct.lean [hunk 2]
```diff
@@ -7,11 +7,17 @@ import Mathlib.Topology.Homeomorph
...
 
-In this file we define topology on `DomMulAct _` and prove basic facts about this topology.
-The topology on `Mᵈᵐᵃ` is the same as the topology on `M`
-(formally, it is induced by `DomMulAct.mk.symm`, since the types aren't definitionally equal).
+In this file we define `TopologicalSpace` structure on `Mᵈᵐᵃ` and `Mᵈᵃᵃ`
+and prove basic theorems about these topologies.
+The topologies on `Mᵈᵐᵃ` and `Mᵃᵐᵃ` are the same as the topology on `M`.
```
Comment (j-loreaux) [inline Mathlib/Topology/Algebra/Constructions/DomMulAct.lean:14]:
```suggestion
The topologies on `Mᵈᵐᵃ` and `Mᵈᵃᵃ` are the same as the topology on `M`.
```

**Review by j-loreaux — COMMENTED** [review]
bors d+

**Comment by urkud** [comment]
bors merge

**Pull request closed without merging**